### DISCUSSION TITLE
Which CUDA image should I use?

### DISCUSSION THREAD

**Discussion by moritztim** [mudler/LocalAI#1519] [General]
https://localai.io/howtos/easy-setup-docker/
Offers no explanation as to what I'm supposed to base my choice on... I have no idea which one I need

**Comment by gitwittidbit** [comment]
+1

**Reply by moritztim** [reply]
https://localai.io/basics/getting_started/
Maybe the info there helps

**Comment by moritztim** [comment]
https://localai.io/basics/getting_started/
There's a bit more info but still not enough
